contribute/creating_package.md

```diff
@@ -0,0 +1,93 @@
+# Creating a Julia package
+
+A detailed page on Julia packages is available [here](http://pkgdocs.julialang.org/v1/). Including the part on [creating packages](http://pkgdocs.julialang.org/v1/creating-packages/).
+
+Julia packages consist of modules. Hence, before creating a package, make sure to understand how [modules](https://docs.julialang.org/en/v1/manual/modules/) work in Julia.
+
+
+Here we provide a step-by-step example, using [PkgTemplates.jl](https://juliahub.com/docs/PkgTemplates/IGiQL/0.6.4/), inspired by the [video](https://www.youtube.com/watch?v=QVmU29rCjaA&t=112s) of Chris Rackauckas.
+
+In case of any issues, remember that [help](https://github.com/elizavetasemenova/www.julialang.org/blob/main/contribute/contributing_for_everyone.md#where-to-get-help) is available.
+
+## What you need before you start
+
+- an [IDE](https://github.com/elizavetasemenova/www.julialang.org/blob/main/contribute/contributing_for_everyone.md#what-are-the-ide-options)
+- a GitHub account
+
+## Starting a package
+
+- create a folder to store your package. We use Atom as an Editor. We created a folder called 'tiddlywinks.jl'. In Atom, we open the GitHub panel, and opt to 'initialize and publish on GitHub'. Now we can go to our GitHub page and see that 'tiddlywinks.jl' has appeared there, as well as locally.
+
+- inside this folder, in the Julia REPL install PkgTemplate.jl with
+```
+using Pkg
+Pkg.add("PkgTemplates")
+```
+
+## Creating package template
+
+Now `PkgTemplates` has been installed and we want to use it to create a template for a package.
+```
+using PkgTemplates
+
+t = Template(;
+           user="elizavetasemenova",
+           license="MIT",
+           authors=["ES"],
+           plugins=[
+               TravisCI(),
+               Codecov(),
+               Coveralls(),
+               AppVeyor(),
+           ],
+       )
+
+t("tiddlywinks")
+```
+
+As we have not specified the folder, the package has been created at `~/.julia/dev` on Linux, and at `username/julia/dev` in Windows. If this is not the repository we want to be working from, let's move the files to the original repo which we have created. Now in `tiddlywinks.jl` we have
```

Comment:
In technical documentation, we usually use second person (you) to address our readers :) I suggest rewriting this paragraph to be consistent with the above ones. See this as reference: https://developers.google.com/style/person

Reply:
really cool document to go over, thanks